### PR TITLE
Feature/demo snag list

### PR DESCRIPTION
**Pull request by torchiad**
Todo:  validation on date

### CODE REVIEW

File: src/components/AllOpportunities/AllOpportunities.tsx [hunk 1]
```diff
@@ -33,10 +36,14 @@ export const AllOpportunities: FC<Props> = ({ ...props }) => (
...
-                                                {opportunity.name}
-                                            </Table.Cell>
-                                            <Table.Cell>
-                                                {opportunity.description}
+                                                <Link
+                                                    to={`/setup/${
+                                                        opportunity.id
+                                                    }`}
+                                                >
+                                                    {" "}
```
Comment (ascorbic) [inline src/components/AllOpportunities/AllOpportunities.tsx]: You can remove these spaces

File: src/components/WebsiteListing/WebsiteListing.tsx [hunk 2]
```diff
@@ -44,7 +56,7 @@ export const WebsiteListing: FC<Props> = ({
     }, []);
 
     if (!websiteListing || !websiteListing.getWebsiteListing) {
-        return <Title>Not found</Title>;
+        return <div> Loading... </div>;
```
Comment (ascorbic) [inline src/components/WebsiteListing/WebsiteListing.tsx]: Do you need to do this? Could we wrap the form with a LoadingBox instead?

**Review by ascorbic — CHANGES_REQUESTED** [review]

File: src/components/WebsiteListing/WebsiteListing.tsx [hunk 3]
```diff
@@ -55,36 +56,32 @@ export const WebsiteListing: FC<Props> = ({
...
-
-    const breadcrumbs = (
-        <Breadcrumbs>
-            <Breadcrumbs.Link href={linkBack}>
-                Opportunity setup
-            </Breadcrumbs.Link>
-            Website listing
-        </Breadcrumbs>
-    );
+    if (!websiteListing) {
```
Comment (ascorbic) [inline src/components/WebsiteListing/WebsiteListing.tsx]: I meant a LoadingBox wrapping the form.

**Review by ascorbic — CHANGES_REQUESTED** [review]

File: src/components/WebsiteListing/WebsiteListing.tsx (hunk 3 above)
Comment (torchiad) [reply src/components/WebsiteListing/WebsiteListing.tsx]: Fixed.

File: src/components/SetupApplication/SetupApplicationForm.tsx [hunk 4]
```diff
@@ -135,12 +113,14 @@ export const SetupApplicationForm: FC<Props> = ({
         const closeDate = new Date(year, month, day, hour, minute);
 
         const today = new Date();
+
         if (openDate < today) {
-            setDateError("The Open date must start after the current date.");
+            setDateError("Open date must be before today");
```
Comment (ascorbic) [inline src/components/SetupApplication/SetupApplicationForm.tsx]: Surely _after_ today

**Review by ascorbic — CHANGES_REQUESTED** [review]

**Review by ascorbic — APPROVED** [review]

**Pull request merged**